qemu/tests/cfg/create_large_raw_img.cfg

```diff
@@ -11,7 +11,8 @@
         - over_xfs:
             file_sys = "xfs"
             image_size_large = 10240000T
-            err_info = "Image size must be less than 8 EiB!"
+            err_info = "Image size must be less than 8 EiB!;"
+            err_info = '${err_info}Invalid image size specified. Must be between 0 and 9223372036854775807.'
```

Comment:
`Invalid image size specified. Must be between 0 and 9223372036854775807.'`
From upsteam code:
https://github.com/qemu/qemu/blob/fd3cd581f9dcd11286daacaa5272e721c65aece8/qemu-img.c#L501-L503
```
        error_report("Invalid %s specified. Must be between %" PRId64
                     " and %" PRId64 ".", name, min, max);
```
seems like the number will be changed by version, arch, or others factors.
I think we could set here as a pattern:
err_info = "Image size must be less than \d+ \w+!;"
like err_info = '${err_info}Invalid image size specified. Must be between \d+ and \d+.'
then re.search(....)

This just my suggestion, finally up to you.

Reply:
I wrote it by qemu-iotests/178.out.qcow2. And parameters min and max were defined as int64_t. We must confirm the value is bewteen 0 and 8 EiB (9223372036854775807).